data-raw/growthchart_metadata_creation.R

```diff
@@ -57,3 +69,8 @@ usethis::use_data(who_bmi_for_age_boys, overwrite = TRUE)
 
 usethis::use_data(who_hc_for_age_girls, overwrite = TRUE)
 usethis::use_data(who_hc_for_age_boys, overwrite = TRUE)
+
+usethis::use_data(cdc_bmiage, overwrite = TRUE)
```

Comment:
@Pierre-Wallet I've mimic'd the way you handled your PR and renamed the metadata creation file to be generalized and contain the cdc datasets too